custom/reach/const.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy as _
```

Comment:
I think these need to be `ugettext_noop` as they are being evaluated before a language is knownw

Reply:
:+1: